packages/openapi-v3/src/types.ts

```diff
@@ -43,6 +43,8 @@ export interface TagsDecoratorMetadata {
   tags: string[];
 }
 
+export type VisibilityDecoratorMetadata = 'documented' | 'undocumented';
```

Comment:
`enum` or union type, which one is better?

Reply:
Personally, I am fine with the union type.

Reply:
`enum` may be useful for pure-JavaScript experiences, but I'm ok with either.

Reply:
Changed to `enum` to be in line with existing packages (e.g. `repository`, `authorization`, context`)